nest_elephant_tvb/translation/mpi_translator.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2020 Forschungszentrum Jülich GmbH and Aix-Marseille Université
+# "Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements; and to You under the Apache License, Version 2.0. "
+import time
+import numpy as np
+from mpi4py import MPI
+from threading import Thread, Lock
+from nest_elephant_tvb.translation.science_nest_to_tvb import store_data,analyse_data
+
+lock_status=Lock() # locker for manage the transfer of data from thread
+
+def init(path, param, comm, comm_receiver, comm_sender, loggers):
+    '''
+    Initialize the translation with MPI. NOTE: more information will be added with more changes. Currently the multithreaded version is still in use.
+    
+    '''
+    
+    # destructure logger list to indivual variables
+    logger_master, logger_receive, logger_send = loggers
+    # science part, see import
+    # TODO: use os.path (or similar) for proper file handling.
+    store = store_data(path+'/log/',param)
+    analyse = analyse_data(path+'/log/',param)
+    # variable for communication between thread
+    status_data=[0] # status of the buffer
+    
+    ############ NEW Code: MPI intracommunicator
+    # receive on rank 0, science and send on rank 1
+    # TODO: future work: mpi parallel, use rank 1-x for science and sending
+    # NOTE: still copy-paste from kim master thesis code....
+    # new server-intracommunicator without receiving rank 0
+    intracomm = comm.Create(comm.Get_group().Excl([0]))
+    # create the shared memory block / databuffer
+    buf_pkgs = 100 # NOTE: size hardcoded!
```

Comment:
POD: the buffer package may be set in configuration settings.

Reply:
Added this to [Issue 3](https://github.com/multiscale-cosim/TVB-NEST/issues/3)